api/tests/routes/test_detection.py

```diff
@@ -16,9 +16,14 @@ async def test_text_detection(test_app_asyncio, mock_detection_image):
     gt_boxes[:, [1, 3]] = gt_boxes[:, [1, 3]] / 2339
 
     # Check that IoU with GT if reasonable
-    assert isinstance(json_response, list) and len(json_response) == gt_boxes.shape[0]
-    pred_boxes = np.array([elt["box"] for elt in json_response])
+    assert isinstance(json_response, list) and len(json_response) == 2
+    first_pred = json_response[0]
+    assert isinstance(first_pred, dict) and len(first_pred["boxes"]) == gt_boxes.shape[0]
+    pred_boxes = np.array(first_pred["boxes"])
     iou_mat = box_iou(gt_boxes, pred_boxes)
     gt_idxs, pred_idxs = linear_sum_assignment(-iou_mat)
     is_kept = iou_mat[gt_idxs, pred_idxs] >= 0.8
     assert gt_idxs[is_kept].shape[0] == gt_boxes.shape[0]
```

Comment:
You can do a for loop here to test all files from your list

Reply:
yeah but i think this makes not much sense because we send the image twice to test the doc building also (so we will get the same result) ^^